### ISSUE TITLE
Font encodeing setting

### ISSUE THREAD

**Issue by kotrs** [d3alek/Texample2#6]
Hi, 
is it possible to set up encoding for the international fonts ?

**Comment by d3alek** [comment]
What issues do you see with international fonts?

**Comment by kotrs** [comment]
I need Czech fonts with accents...so I modified the range of character_end to 253...so I got all chars, but the font uses 8859-2 and some chars are incorrect. I need to know, how can I change encoding...TypeFace, Paint and Canvas classes have no charset method.

**Comment by d3alek** [comment]
You are right - my suggestion would be to try using a different font than the provided Roboto-Regular.ttf and see if that works.

**Comment by kotrs** [comment]
I tried czech fonts, but Android environment encoding has set different encoding and some chars are displayed incorrectly.
